tests/client.test.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+require('should');
+
+var benv = require('benv');
+
+describe('client', function ( ) {
+
+  before(function (done) {
+    benv.setup(function() {
+      benv.expose({
+        $: require('jquery')
+        , jQuery: require('jquery')
+        , d3: require('d3')
+        , io: {
+          connect: function mockConnect() {
+            return {
+              on: function mockOn(event, callback) {}
+            };
+          }
+        }
+      });
+//      benv.require('../static/js/ui-utils.js', 'window');
+//      benv.require('../static/bower_components/tipsy-jmalonzo/src/javascripts/jquery.tipsy.js');
+      done();
+    });
+  });
+
+  after(function (done) {
+    benv.teardown();
+    done();
+  });
+
+  it ('not blow up', function () {
+//    var jsdom = require('jsdom').jsdom;
+//    var doc = jsdom('<body></body>');
+//    var window = doc.parentWindow;
+//
+//    global.$ = global.jQuery = require('jquery')(window);
+
+    var serverSettings = {
+      apiEnabled: true, careportalEnabled: true, enabledOptions: "ar2 careportal delta direction upbat errorcodes", defaults: {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/799258325/issues/source?bid=2452379&fileBranchId=2725007#l42)